gapis/resolve/command_tree.go

```diff
@@ -91,6 +91,7 @@ func CommandTreeNode(ctx context.Context, c *path.CommandTreeNode) (*service.Com
 			NumChildren: item.Count(),
 			Commands:    cmdTree.path.Capture.CommandRange(uint64(item.Range.First()), uint64(item.Range.Last())),
 			Group:       item.Name,
+			NumCommands: item.DeepCount(func(g atom.Group) bool { return true; /* TODO: Subcommands */ }),
```

Comment:
I figured that since most of our hierarchies are only a couple of levels deep (frame->draw call), this is preferable than the memory overhead to cache it.